plugins/myfitnesspal.py

```diff
@@ -45,7 +57,7 @@ def mfp(text, bot):
             output += ("{caption}: {total}/{remain}{units} ({pct}%) "
                        .format(**kwargs))
 
-        output += " ({})".format(scrape_url.format(text))
+        output += " ({})".format(scrape_url.format(user, date))
```

Comment:
This should probably use `request.url` instead of duplicating the URL generation code.